### PR TITLE
Remove common (redundant) dependencies

### PR DESCRIPTION
**Pull request by tylerbenson**
These dependencies are provided by the parent (instrumentation) project, and don't need to be manually added each time.

### CODE REVIEW

File: dd-java-agent/instrumentation/vertx/vertx.gradle [hunk 1]
```diff
@@ -32,16 +32,8 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-  compileOnly group: 'io.vertx', name: 'vertx-web', version: '3.5.0'
+//  compileOnly group: 'io.vertx', name: 'vertx-web', version: '3.5.0'
```
Comment (randomanderson) [inline dd-java-agent/instrumentation/vertx/vertx.gradle]: Was this intentional?  If it was, it should probably be in a different pull request.

Comment (tylerbenson) [reply dd-java-agent/instrumentation/vertx/vertx.gradle]: Yes.  That folder doesn't actually have any instrumentation, so we don't need to have a compile dependency there.  I left it commented out assuming we would need instrumentation in the near future.
